client/spec/send_spec.js

```diff
@@ -70,4 +76,26 @@ describe('send', function() {
         expect(monitoring.getGroups().count()).toBe(6);
     });
 
+    it('can submit item to a desk although there are spelling mistakes', function () {
+        workspace.open();
+        workspace.editItem(1);
+
+        //Skip spell check
+        authoring.toggleAutoSpellCheck();
+        expect(element(by.model('spellcheckMenu.isAuto')).getAttribute('checked')).toBeFalsy();
+
+        authoring.writeText('Text, that not saved yet');
+        authoring.sendTo('Sports Desk');
+        element(by.className('modal-content')).all(by.css('[ng-click="ok()"]')).click();
```

Comment:
This is repeated multiple times. Can we assign it to a variable in it's helper class?